### DISCUSSION TITLE
Q:请问密码是什么？

### DISCUSSION THREAD

**Discussion by N55743** [LingmoOS/Community#4] [问题解答] [answered]
请问密码是什么？
![image](https://github.com/user-attachments/assets/f6eb3f2c-bbcf-437b-a1e9-969fccdd0f7b)
请问密码是什么？
![image](https://github.com/user-attachments/assets/f6eb3f2c-bbcf-437b-a1e9-969fccdd0f7b)
初次安装系统时他要求我输密码，但我并不知到密码是什么。

_Originally posted by @N55743 in https://github.com/LingmoOS/Community/discussions/1#discussioncomment-10175101_
_Originally posted by @N55743 in https://github.com/LingmoOS/Community/discussions/1#discussioncomment-10175101_

**Comment by elysia-best** [comment, marked as answer]
密码为 live
